src/main/scripts/org/reaktivity/specification/nukleus/tcp/streams/rfc793/client.sent.reset.and.end/server.rpt

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property newClientAcceptRef ${nuklei:newReferenceId()} 
+property newServerConnectRef ${newClientAcceptRef} # external scope
+
+property serverConnect "nukleus://target/streams/tcp#source"
+       
+accept ${serverConnect}
+       option nukleus:route ${newServerConnectRef}
+       option nukleus:window 8192
+       option nukleus:transmission "duplex"
+accepted
+
+connected
+
+write "some data"
```

Comment:
Please remove the writing of data, that's not what this test is targeting

Reply:
Done.